### ISSUE TITLE
installation error on Laravel 6.20.44 with PHP 8.0.18

### ISSUE THREAD

**Issue by handhikadj** [staudenmeir/laravel-adjacency-list#104]
`composer require staudenmeir/laravel-adjacency-list`
                                                                                                                                                       
```
Info from https://repo.packagist.org: #StandWithUkraine
Using version ^1.10 for staudenmeir/laravel-adjacency-list
./composer.json has been updated
Running composer update staudenmeir/laravel-adjacency-list
Loading composer repositories with package information
Info from https://repo.packagist.org: #StandWithUkraine
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - staudenmeir/laravel-adjacency-list[v1.10, ..., v1.10.3] require illuminate/database ^9.0 -> found illuminate/database[v9.0.0-beta.1, ..., 9.x-dev] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires staudenmeir/laravel-adjacency-list ^1.10 -> satisfiable by staudenmeir/laravel-adjacency-list[v1.10, v1.10.1, v1.10.2, v1.10.3].


Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

`composer require staudenmeir/laravel-adjacency-list:"^1.0"`

```
Problem 1
    - staudenmeir/laravel-adjacency-list[v1.0, ..., 1.0.x-dev] require illuminate/database ~5.5.29|5.6.*|5.7.* -> found illuminate/database[v5.5.33, ..., 5.7.x-dev] but these were not loaded, likely because it conflicts with another require.
    - staudenmeir/laravel-adjacency-list[v1.1, ..., 1.1.x-dev] require php ^7.1.3 -> your php version (8.0.18) does not satisfy that requirement.
    - staudenmeir/laravel-adjacency-list[v1.2, ..., 1.3.x-dev] require php ^7.2 -> your php version (8.0.18) does not satisfy that requirement.
    - staudenmeir/laravel-adjacency-list[v1.4, ..., 1.5.x-dev] require php ^7.2.5 -> your php version (8.0.18) does not satisfy that requirement.
    - staudenmeir/laravel-adjacency-list[v1.6.1, ..., 1.9.x-dev] require illuminate/database ^8.0 -> found illuminate/database[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - staudenmeir/laravel-adjacency-list v1.6 requires php ^7.3 -> your php version (8.0.18) does not satisfy that requirement.
    - staudenmeir/laravel-adjacency-list[v1.10, ..., v1.10.3] require illuminate/database ^9.0 -> found illuminate/database[v9.0.0-beta.1, ..., 9.x-dev] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires staudenmeir/laravel-adjacency-list ^1.0 -> satisfiable by staudenmeir/laravel-adjacency-list[v1.0, ..., v1.10.3].
```

**Comment by staudenmeir** [comment]
What operating system are you using?

**Comment by handhikadj** [comment]
mac m1

**Comment by staudenmeir** [comment]
What terminal are you using? Are you in an IDE?

**Comment by handhikadj** [comment]
iterm

**Comment by staudenmeir** [comment]
I thought it was a terminal issue, but the package didn't support PHP 8 on Laravel 6. I released `v1.3.3` that brings support for PHP 8 to Laravel 6.
